src/System.Collections/ref/System.Collections.cs

```diff
@@ -107,8 +107,6 @@ public partial class Dictionary<TKey, TValue> : System.Collections.Generic.IColl
         void System.Collections.IDictionary.Remove(object key) { }
         System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
         public bool TryGetValue(TKey key, out TValue value) { throw null; }
-        public TValue GetValueOrDefault(TKey key) { throw null; }
-        public TValue GetValueOrDefault(TKey key, TValue defaultValue) { throw null; }
```

Comment:
GetValueOrDefault is an extension Method for IDictionary that is why we are removing it so the tests would still pass. I'm verifying desktop tests though.